six modules removed during cleanup

[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

<!--- If you are new to Docker or this application our issue tracker is **ONLY** used for reporting bugs or requesting features. Please use [our discord server](https://discord.gg/YWrKVTn) for general support. --->

<!--- If this acts as a feature request please ask yourself if this modification is something the whole userbase will benefit from --->
<!--- If this is a specific change for corner case functionality or plugins please look at making a Docker Mod or local script  https://blog.linuxserver.io/2019/09/14/customizing-our-containers/ -->

<!--- Provide a general summary of the issue in the Title above -->

------------------------------

## Expected Behavior
Apprise would work from NZBGET. Notify script visible in settings NZBGET

## Current Behavior
Module six is removed
https://github.com/linuxserver/docker-nzbget/blob/master/Dockerfile#L96


## Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1. docker exec -it nzbget /bin/bash
2. python Notify.py
3.
4.

## Environment
**OS:** Debian 10.6
**CPU architecture:** x86_64
**How docker service was installed:** official docker repo


## Command used to create docker container (run/create/compose/screenshot)
```
version: "2"
services:
  nzbget:
    image: linuxserver/nzbget
    container_name: nzbget
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Europe/Amsterdam
      - DOCKER_MODS=linuxserver/mods:universal-apprise
    volumes:
      - /home/<userid>/docker-config/nzbget/config:/config
      - /mnt/storage/tempdata/complete:/downloads
    ports:
      - 6789:6789
    restart: unless-stopped
```

## Docker logs
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 01-envfile: executing...
[cont-init.d] 01-envfile: exited 0.
[cont-init.d] 10-adduser: executing...

-------------------------------------
          _         ()
         | |  ___   _    __
         | | / __| | |  /  \
         | | \__ \ | | | () |
         |_| |___/ |_|  \__/


Brought to you by linuxserver.io
-------------------------------------

To support the app dev(s) visit:
NZBGet: https://nzbget.net/donate

To support LSIO projects visit:
https://www.linuxserver.io/donate/
-------------------------------------
GID/UID
-------------------------------------

User uid:    1000
User gid:    1000
-------------------------------------

[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 30-config: executing...
[cont-init.d] 30-config: exited 0.
[cont-init.d] 99-custom-files: executing...
[custom-init] no custom files found exiting...
[cont-init.d] 99-custom-files: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[INFO] nzbget 21.0 server-mode
[INFO] Reloading...
[INFO] nzbget 21.0 server-mode
[INFO] Executing script Notify.py with command TestServers
[INFO] Notify: Traceback (most recent call last):
[INFO] Notify:   File "/downloads/scripts/Notify.py", line 169, in <module>
[INFO] Notify:     import six
[INFO] Notify: ModuleNotFoundError: No module named 'six'
[ERROR] Script Notify.py with command TestServers failed (terminated with unknown status)
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -78,8 +78,8 @@ RUN \
 	libxslt \
 	openssl \
 	p7zip \
-	python3 \
 	py3-pip \
+	python3 \
 	unrar \
 	wget && \
  echo "**** install python packages ****" && \

--- a/Dockerfile
+++ b/Dockerfile
@@ -70,7 +70,6 @@ RUN \
 	libxml2-dev \
 	libxslt-dev \
 	make \
-	py3-pip \
 	python3-dev && \
  echo "**** install packages ****" && \
  apk add --no-cache \
@@ -80,6 +79,7 @@ RUN \
 	openssl \
 	p7zip \
 	python3 \
+	py3-pip \
 	unrar \
 	wget && \
  echo "**** install python packages ****" && \

--- a/Dockerfile.aarch64
+++ b/Dockerfile.aarch64
@@ -78,8 +78,8 @@ RUN \
 	libxslt \
 	openssl \
 	p7zip \
-	python3 \
 	py3-pip \
+	python3 \
 	unrar \
 	wget && \
  echo "**** install python packages ****" && \

--- a/Dockerfile.aarch64
+++ b/Dockerfile.aarch64
@@ -70,7 +70,6 @@ RUN \
 	libxml2-dev \
 	libxslt-dev \
 	make \
-	py3-pip \
 	python3-dev && \
  echo "**** install packages ****" && \
  apk add --no-cache \
@@ -80,6 +79,7 @@ RUN \
 	openssl \
 	p7zip \
 	python3 \
+	py3-pip \
 	unrar \
 	wget && \
  echo "**** install python packages ****" && \

--- a/Dockerfile.armhf
+++ b/Dockerfile.armhf
@@ -78,8 +78,8 @@ RUN \
 	libxslt \
 	openssl \
 	p7zip \
-	python3 \
 	py3-pip \
+	python3 \
 	unrar \
 	wget && \
  echo "**** install python packages ****" && \

--- a/Dockerfile.armhf
+++ b/Dockerfile.armhf
@@ -70,7 +70,6 @@ RUN \
 	libxml2-dev \
 	libxslt-dev \
 	make \
-	py3-pip \
 	python3-dev && \
  echo "**** install packages ****" && \
  apk add --no-cache \
@@ -80,6 +79,7 @@ RUN \
 	openssl \
 	p7zip \
 	python3 \
+	py3-pip \
 	unrar \
 	wget && \
  echo "**** install python packages ****" && \

--- a/README.md
+++ b/README.md
@@ -247,6 +247,7 @@ Once registered you can define the dockerfile to use with `-f Dockerfile.aarch64
 
 ## Versions
 
+* **26.10.20:** - Fix python dependencies.
 * **24.08.20:** - Fix ignored umask environment variable.
 * **08.06.20:** - Symlink python3 bin to python.
 * **01.06.20:** - Rebasing to alpine 3.12. Removing python2.

--- a/readme-vars.yml
+++ b/readme-vars.yml
@@ -70,6 +70,7 @@ app_setup_block: |
 
 # changelog
 changelogs:
+  - { date: "26.10.20:", desc: "Fix python dependencies." }
   - { date: "24.08.20:", desc: "Fix ignored umask environment variable." }
   - { date: "08.06.20:", desc: "Symlink python3 bin to python." }
   - { date: "01.06.20:", desc: "Rebasing to alpine 3.12. Removing python2." }

